models/public/face-detection-retail-0044/face-detection-retail-0044.md

```diff
@@ -0,0 +1,56 @@
+# face-detection-retail-0044
```

Comment:
Isn't this a duplicate file?

Reply:
@IRDonch no, the file above is a duplicate

Reply:
Sure, all I'm saying is that there's some duplication going on. :-)

Reply:
Thanks, it's an old version of file, forgot to delete